### PR TITLE
Feat: Add unPinMessage Method & pinMessage Fix

### PR DESCRIPTION
**Pull request by bananacity**
added unpin endpoint to remove whatever message is currently pinned

### CODE REVIEW

**Comment by zSoulweaver** [comment]
Lgtm.

I'm currently away on holiday, when I am back, towards the end of next week, I'll merge it in and release a new ver.

Thanks!

**Comment by bananacity** [comment]
I also fixed the pinMessage endpoint, it unnecesarily stringified the body causing a 422 error with kick. works with this fix now

File: src/endpoints/chat/chat.endpoint.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ export class ChatEndpoint extends BaseEndpoint {
       endpoint: `api/v2/channels/${channel}/pinned-message`,
       method: 'post',
       options: {
-        body: JSON.stringify(body),
+        body: body,
```
Comment (zSoulweaver) [inline src/endpoints/chat/chat.endpoint.ts:95]: Could we shorthand this to just `body,`.

File: src/endpoints/chat/chat.endpoint.ts [hunk 2]
```diff
@@ -105,6 +105,23 @@ export class ChatEndpoint extends BaseEndpoint {
     return deserializedResponse
   }
 
+  public async unPinMessage(channel: string) {
```
Comment (zSoulweaver) [inline src/endpoints/chat/chat.endpoint.ts:108]: Unpin is one word, could we match that here as well, `unpinMessage`.

**Review by zSoulweaver — CHANGES_REQUESTED** [review]
Just a few nitpicky things is we could fix those up. Otherwise I've tested everything and it seems to work well. :)

**Pull request merged**